session-manager/common/session/Session.cpp

```diff
@@ -324,6 +355,22 @@ namespace ogon { namespace sessionmanager { namespace session {
 			}
 		}
 
+		if (!SetEnvironmentVariableEBA(&mpEnvBlock, "PULSE_CONFIG_PATH", pulseconfigpath.c_str())) {
+			WLog_Print(logger_Session, WLOG_ERROR, "s %" PRIu32 ": failed to set PULSE_CONFIG_PATH in the environment block", mSessionID);
+			return false;
+		}
```

Comment:
This could easily be set via `environment_add_string`https://github.com/ogon-project/ogon/blob/master/doc/config.md#environment_add_string

Reply:
Same here: code "worked for me" - cleanups by someone more familiar with ogon are welcome.
However I think, there may have been a difference between the session environment and the environment used for virtual channels scripts ... but I'm not sure about this (last tests were half a year ago).